CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Search Builder - Fix broken queries on ObjectIDs and Numbers.
```

Comment:
Let's consider this as a feature 👍 => `Added` section
`Search builder` is very technical, we should focus on the end user experience. => "Search"
 I would add 2 lines to identify easily the 2 things you added

```
### Added
- Search - Add the search on ObjectID fields (exact match only). 
- Search - Add the search on Number fields (exact match only).
```

Reply:
Ok, got your point! Thanks 💯